administrator/components/com_categories/Field/Modal/CategoryField.php

```diff
@@ -210,7 +205,7 @@ function jSelectCategory_" . $this->id . "(id, title, object) {
 					'width'       => '800px',
 					'bodyHeight'  => 70,
 					'modalWidth'  => 80,
-					'footer'      => '<a role="button" class="btn btn-secondary" data-dismiss="modal" aria-hidden="true">'
+					'footer'      => '<a role="button" id="category-modal-select" class="btn btn-secondary" data-dismiss="modal" aria-hidden="true" data-select-id=' . $this->id . '>'
```

Comment:
Line exceeds 150 characters; contains 167 characters (thats what phpcs is complaining about)